h2o-core/src/main/java/hex/ScoreKeeper.java

```diff
@@ -466,6 +469,7 @@ public String toString() {
         ", _hitratio=" + Arrays.toString(_hitratio) +
         ", _lift=" + _lift +
         ", _anomaly_score_normalized=" + _anomaly_score_normalized +
+        ", _custom=" + _custom_metric +
```

Comment:
Should the text correspond to the field name? I.e. 
` ", _custom_metric=" + _custom_metric +`

Reply:
yeah, sure - it is better for consistency to keep the print out the same - will fix, thank you

Reply:
fixed